README.md

```diff
@@ -250,28 +250,12 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // If provided, it MUST be base64url encoded.
                 "access_token_secret": "xxxxxxxxxxxxxxx",
 
-                // The secret used when validating an OpenID Connect ID Token
-                // signed with a symmetric algorithm (e.g. HS256).
-                // Used if provided, otherwise, defaults to validating
-                // with "client_secret".
-                // If provided, it MUST be base64url encoded.
-                "nonstandard_id_token_secret": "xxxxxxxxxxxxxxx",
-
                 // Controls whether the HTTP API requires the presence of the
                 // "aud" member in the introspection endpoint response. If set
                 // to true and the "aud" member is NOT present, the provided
                 // access token will be rejected.
                 "require_aud_member_from_introspection_endpoint": false,
 
-                // The OIDC mode the HTTP API will run as.
-                // The following values are supported:
-                // - client:              Run the HTTP API as an OIDC client
-                // - protected_resource:  Run as an OAuth Protected Resource
-                "mode": "client",
-
-                // URI pointing to the irods HTTP API auth endpoint.
-                "redirect_uri": "https://<domain>/irods-http-api/<version>/authenticate",
```

Comment:
I believe there's at least one other reference to redirect_uri in the README. Please check for other instances of removed config options.

Reply:
Removed.